flake.nix

```diff
@@ -76,7 +76,7 @@
             }
             // lib.optionalAttrs (config.compiler-nix-name == defaultCompiler) {
               # tools that work only with default compiler
-              haskell-language-server.src = nixpkgs.haskell-nix.sources."hls-2.6";
+              haskell-language-server.src = inputs.patched-hls;
```

Comment:
Why not use HLS 2.8 rather than patched version?

Reply:
@carbolymer> [HLS 2.8](https://github.com/haskell/haskell-language-server/releases/tag/2.8.0.0) doesn't support the Nix shell's current GHC version (9.6.4)

Reply:
can we bump nix shell to 9.8?

Reply:
I don't know how to do that, and I am happy to investigate. But I would say getting a quick fix out for the hanging as soon as possible would be beneficial nevertheless

Reply:
https://github.com/IntersectMBO/cardano-cli/blob/main/flake.nix#L50 - you can set it to `"ghc982"`. I think I'd avoid adding an input to the flake if possible.

Reply:
Ok, I've done that now instead, it seems to have worked